tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs

```diff
@@ -144,6 +144,24 @@ public static Result IsAwsSqsV0(this MockSpan span) => Result.FromSpan(span)
                 .Matches("component", "aws-sdk")
                 .Matches("span.kind", "client"));
```

Comment:
I think you need to add the following declarations to the Sqs rules (both v0 and v1):
- IsOptional("aws_service")
- IsOptional("queuename")

Reply:
Or, if those tags are no longer added, then perhaps the snapshot files need to be updated